### ISSUE TITLE
Update to Phosphors 2.0

### ISSUE THREAD

**Issue by ivanchaukn** [afkcodes/flutter_phosphor_icons#7]
It will be appreciated if we can upgrade this package to use Phosphors Icons 2.0

**Comment by afkcodes** [comment]
hey @ivanchaukn  thanks for raising the issue this update will come tomorrow, was busy this week.

**Comment by afkcodes** [comment]
seems like the update is a pretty major one there are lot of changes to it , this may take some time @ivanchaukn

**Comment by Rosovskyy** [comment]
@afkcodes Hey! Don't wanna rush you, but is it gonna be merged in the nearest future?

**Comment by afkcodes** [comment]
@Rosovskyy yes  it will be, i'm just occupied with some personal health stuff

**Comment by danielkauffmann** [comment]
any updates guys?

**Comment by afkcodes** [comment]
@danielkauffmann hello thanks for using the package, Im sorry that i am not able to work on this, as my health is not on the track. will try to work on this once i feel better. hope you understand. and thanks again for using this package

**Comment by danielkauffmann** [comment]
Hey @afkcodes, thank you very much for the feedback, and I apologize for my rudeness. I genuinely hope everything goes well for you. Sometimes, us devs, have this anxiety for updates that seem to take long, but our health cannot be disregarded, and I completely understand you. Thank you for your contribution to this repository, and I'm rooting for all of this to pass!
